splinterd/src/config/partial.rs

```diff
@@ -0,0 +1,338 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[cfg(not(feature = "config-toml"))]
+use std::fs::File;
+#[cfg(not(feature = "config-toml"))]
+use std::io::Read;
+
+#[cfg(not(feature = "config-toml"))]
+use toml;
+
+#[cfg(not(feature = "config-toml"))]
+use crate::config::ConfigError;
+
+/// PartialConfig is an object to be used as an intermediary while collecting the config values
```

Comment:
Would suggest "PartialConfig is an intermediate representation of configuration values, used when combining several sources." as the opening sentence.

Reply:
That's definitely better wording, I changed the sentence to your suggestion.